contrib/mathtex-script-type/README.md

```diff
@@ -0,0 +1,16 @@
+# `math/tex` Custom Script Type Extension
+
+This is an extension to automatically display code inside `script` tags with `type=math/tex` using KaTeX.
+This script type is commonly used by MathJax, so this can be used to support compatibility with MathJax.
+
+### Usage
+
+This extension isn't part of KaTeX proper, so the script should be separately
+included in the page.
+This include should be loaded after all `script` blocks you want to render.
+
+```html
+<script src="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.7.1/contrib/mathtex-script-type.min.js" integrity="sha384-o+v+EkJWQmZj7XwHBxehTGJKE18182WyyN2glZMTPw9g5XxjN1uwrquNuMX/NJiF"></script>
```

Comment:
Are you suggesting that the Makefile automatically tweak the README to include the latest version number?  This seems like it'd be cool, though I don't quite know how it'd work...

Perhaps a simpler solution would be to write `<script src=".../KaTeX/contrib/mathtex-script-type.min.js"></script>` and let users fill in the right path, either local to their webserver (as many are probably using it) or on CDN?  We could also give a template for CDN including, but with `...` or something else in place of the actual version number.